mongodb/src/main/java/com/yahoo/ycsb/db/OptionsSupport.java

```diff
@@ -76,7 +76,7 @@ else if ("acknowledged".equals(writeConcernType)) {
                 result = addUrlOption(result, "w", "1");
             }
             else if ("journaled".equals(writeConcernType)) {
-                result = addUrlOption(result, "journal", "true");
+                result = addUrlOption(result, "j", "true");
```

Comment:
Oh.. Hummm. The problem is that the async driver only supports the "journal" parameter. Since "journal" is the documented name I think we should leave that. I wouldn't be opposed to adding both a "journal" and "j" parameter to the URI with a reference to remove it once the Jira ticket is released.